src/interface/variable.hpp

```diff
@@ -80,14 +94,14 @@ class CellVariable {
   KOKKOS_FORCEINLINE_FUNCTION
   auto GetDim(const int i) const {
     // we can't query data.GetDim() here because data may be unallocated
-    assert(0 < i && i <= 6 && "ParArrayNDGenerics are max 6D");
+    assert(0 < i && i <= 6 && "ParArrayNDs are max 6D");
```

Comment:
I think this still only supports 6D since `ParArrayND` is still 6D underneath the hood. Of course, it would only be a couple of lines to change this.

Reply:
Ah, I thought I saw that `ParArrayND` was 7D under the hood now. Maybe not.